app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.kt

```diff
@@ -33,16 +41,26 @@ class LoginFragment private constructor() : WolmoFragment<FragmentLoginBinding,
         binding.edtPassword.error = getString(R.string.passwrod_requerided)
     }
 
-    // For test
-    override fun showSuccessMessage() {
-        Toast.makeText(context, getString(R.string.success), Toast.LENGTH_SHORT).show()
-    }
-
     override fun setCredentials(username: String, password: String) {
         binding.edtEmail.setText(username)
         binding.edtPassword.setText(password)
     }
 
+    override fun showTermsAndConditions() {
+        val intent = Intent(Intent.ACTION_VIEW, Uri.parse(Extras.Const.URL_WOLOX))
+        startActivity(intent)
+    }
+
+    override fun goToHome() {
+        val intent = Intent(context, HomeActivity::class.java)
+        startActivity(intent)
+    }
+
+    override fun goToSignUp() {
+        val intent = Intent(context, SignUpActivity::class.java)
+        startActivity(intent)
+    }
```

Comment:
You can create a static method in SignUpActivity and HomeActivity named `start()` that does this, so you just need to call `HomeActivity.start(context)`